.github/workflows/build.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     strategy:
       matrix:
         version:
-            - "StableFullRelease"
-            - "StableFdroidRelease"
+            - "OoniStableFullRelease"
+            - "OoniStableFdroidRelease"
```

Comment:
TODO(@aanorbel): please, update the main `README.md` given that it contains build instructions and lists all the variants as an introduction to the reader.

TODO(@bassosimone): make sure we also update the monorepo scripts once we merge.